libraries/hatch-user-management-client/src/constants.ts

```diff
@@ -0,0 +1 @@
+export const WEB_SERVICES_URL = process.env.WEB_SERVICES_URL;
```

Comment:
Per above, lets inject this specifically for AUTH -- we might use different base URLs for auth vs. other services.

Reply:
removed